doc/user-guide/src/installation/other.md

```diff
@@ -74,6 +74,17 @@ and you should be able to run `rustup`, `rustc`, `cargo`, etc. normally.
 
 [correctly exposed]: ./already-installed-rust.html#ensure-the-correct-path-configuration
 
+### winget
+
+`MSVC` Pre Requisite
```

Comment:
The MSVC installation instructions belong to another section which is located under https://rust-lang.github.io/rustup/installation/windows-msvc.html (you can use relative links to refer to it if you want) so please don't put it here.